cdg-pitest-license.txt

```diff
@@ -0,0 +1,7 @@
+#License file for pitest extensions
+#Mon Nov 07 08:41:17 GMT 2022
+expires=07/11/2023
+keyVersion=1
+signature=MhZxMbnO6UovNfllM0JuVWkZyvRT3/G5o/uT0Mm36c7200VpZNVu03gTAGivnl9W5RzvZhfpIHccuQ5ctjQkrqhsFSrl4fyqPqu3y5V2fsHIdFXP/G72EGj6Kay9ndLpaEHalqE0bEwxdnHMzEYq5y3O9vUPv8MhUl57xk+rvBo\=
+packages=tech.picnic.errorprone.*
+type=OSSS
```

Comment:
@rickie following your last commit the codes doesn't work anymore; the plugin looks for the exact original file. Now all analysis is skipped. From the logs:
```
Licenced for evaluation and demonstration purposes only.
Expires: 2023-09-20
For use with packages: com.example.*
```